light-client/src/prelude.rs

```diff
@@ -13,3 +13,7 @@ pub use crate::{
     store::{memory::*, sled::*, LightStore, VerifiedStatus},
     types::*,
 };
+
+pub fn todo<A>() -> A {
+    unreachable!()
+}
```

Comment:
Is this used anywhere?

Reply:
Not anymore, thanks for pointing it out!